docs/api/instruments/ametek/ametek7270.rst

```diff
@@ -0,0 +1,7 @@
+################################
+Ametek 7270 DSP Lockin Amplifier
+################################
+
+.. autoclass:: pymeasure.instruments.danfysik.Danfysik8500

```

Comment:
This should be referencing `pymeasure.instruments.ametek.Ametek7270`.

Reply:
Whoops! that's what I get for copy/pasting. It should be fixed now.